README.md

```diff
@@ -70,6 +71,7 @@ p => port forward pod
 l => log popup
 e => exec popup
 j => jab deployment to force rolling update
+S => scale resource replicas
```

Comment:
is it `S` or `s`?

Reply:
Right, I'll change it to use uppercase s in all code, including transient.